src/test/java/com/intive/patronage22/szczecin/retroboard/service/BoardServiceTest.java

```diff
@@ -457,7 +457,8 @@ void assignUsersToBoardShouldReturnFailedEmails() throws JSONException {
         final String displayName = "testDisplayName";
         final User owner = new User("123", ownerEmail, displayName, Set.of());
         final User user = new User("1234", usersEmails.get(0), displayName, new HashSet<>());
-        final Board board = buildBoard(owner);
+        final var state = EnumStateDto.CREATED;
```

Comment:
you don't need to create state as separate variable if it's used only in line below
```suggestion
        final var board = buildBoard(user, EnumStateDto.CREATED);
```

Reply:
done